code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -21,6 +24,26 @@
 
 	if(src.stat == 2.0 && (act != "deathgasp"))
 		return
+
+	switch(act)
```

Comment:
the actual emote path/if branch needs to handle these. doing it like this is unmaintainable.